vscode/src/chat/chat-view/ChatPanelsManager.ts

```diff
@@ -165,6 +165,15 @@ export class ChatPanelsManager implements vscode.Disposable {
         }
     }
 
+    public async editChatHistory(chatID: string): Promise<void> {
+        await vscode.window.showInputBox({ prompt: 'Enter new chat history name' }).then(async message => {
```

Comment:
```suggestion
        await vscode.window.showInputBox({ prompt: 'Enter new chat name' }).then(async message => {
```

Could we also pass in the current title as `value`?

Reply:
Yes, we can pass that @toolmantim. I will implement this.